vendor/brave-ios/Ads/BATBraveAds.mm

```diff
@@ -905,6 +916,11 @@ - (void)runDBTransaction:(ads::DBTransactionPtr)transaction
   }
 }
 
+- (void)onAdRewardsChanged
+{
+  // TODO(brave): Refresh Brave Ads UI
```

Comment:
What UI in particular? How is this getting to client? Should be included in this PR and brave-ios PR

Reply:
Fixed